source/api/presentation/3.0/index.md

```diff
@@ -737,17 +737,31 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id` and
 
 A link from this Manifest or Range, to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases. When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Range. This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually navigate to find it.
 
-The value _MUST_ be a JSON object, which _MUST_ have the `id` and `type` properties.  The object _MUST_ be either a Canvas, or a Specific Resource with a Selector and a `source` property where the value is a Canvas.
+The value _MUST_ be a JSON object, which _MUST_ have the `id` and `type` properties.  The object _MUST_ be either a Canvas (example 1), or a Specific Resource with a Selector and a `source` property where the value is a Canvas (example 2).
```

Comment:
Since the examples aren't numbered, should we say "first example below" and "second example below" instead of "example 1" etc. ?

Reply:
True. Will fix.